spec/token_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../lib/token'
+
+describe 'Token' do
+    include Token
+    context 'gets tokens' do
+        it '- player 1' do
```

Comment:
The descriptions are a bit weird here. I know this reads nice when you include "context" and "it" together, but the description for "it" seems incomplete